src/components/console/RequestList/Item.js

```diff
@@ -0,0 +1,40 @@
+import { useStyles } from './Item.style'
+import { useConsoleContext } from 'src/context/ConsoleContext'
+import React from 'react'
```

Comment:
It's better to firsly put main imports as React, etc, then less important and then all others. In your case firstly import React, then hooks, functions, styles and all the rest on needed stuff.

Reply:
this comment is done